### PR TITLE
Bert nn-Graph based on lazy version

### PR DESCRIPTION
**Pull request by L1aoXingyu**
本 PR 要解决的：
- [x] ofrecord dataloader
- [x] 基于 lazy 版本的 bert module 实现
- [x] 对齐 lazy 版本和 graph 版本前 1000 iter 的 loss

### CODE REVIEW

File: bert-oneflow/tests/test_fc.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    lazy_res, lazy_weight, lazy_bias = fc_job(input_blobs).get()
+
+    # eager fully connected layer
+    eager_fc = nn.Linear(10, 4)
+    eager_fc.weight = nn.Parameter(oneflow.tensor(lazy_weight.numpy().transpose()))
+    eager_fc.bias = nn.Parameter(oneflow.tensor(lazy_bias.numpy()))
+
+    eager_res = eager_fc(oneflow.tensor(input_blobs, dtype=oneflow.float32))
+
+    assert np.allclose(lazy_res.numpy(), eager_res.numpy(), rtol=1e-4, atol=1e-4)
```
Comment (L1aoXingyu) [inline bert-oneflow/tests/test_fc.py:53]: 一个简单的对齐测试

**Comment by CPFLAME** [comment]
### 测试loss对比实验

#### 实验配置

- batch size: 32
- iter_num: 1000
- lr: 5e-4
- scheduler: None
- optimizer：SGD, momentum=0.9

#### bert  lazy和graph对比
compare_lazy_graph_loss
compare_lazy_graph_abs_loss

#### lazy和eager对比
compare_eager_loss
compare_eager_abs_loss

**Comment by L1aoXingyu** [comment]
对齐 1000 iters loss，使用 Adam optimizer，关掉 dataloader random shuffle，所有模型使用相同的初始化

### 实验配置
batch size: 32
iter_num: 1000
lr: 1e-3
scheduler: None
optimizer：Adam

eager_graph_loss
lazy_graph_loss
